src/app/organizations/settings/account.component.html

```diff
@@ -49,9 +49,13 @@ <h1>{{'apiKey' | i18n}}</h1>
     <h1>{{'taxInformation' | i18n}}</h1>
 </div>
 <p>{{'taxInformationDesc' | i18n}}</p>
-<a href="https://bitwarden.com/contact/" target="_blank" rel="noopener" class="btn btn-outline-secondary">
-    {{'contactSupport' | i18n}}
-</a>
+<form *ngIf="org && !loading" #formTax (ngSubmit)="submitTaxInfo()" [appApiAction]="formPromise" ngNativeValidate>
```

Comment:
If we're not going to show anything in this section while things are loading, I would recommend showing a spinner in some way instead of nothing.